finetuner/__init__.py

```diff
@@ -34,77 +36,82 @@ def login():
     ft.login()
 
 
-def list_models():
+def list_callbacks() -> Dict[str, callback.CallbackStubType]:
+    """List available callbacks."""
+    return {
+        name: obj for name, obj in inspect.getmembers(callback) if inspect.isclass(obj)
+    }
+
+
+def _list_models() -> Dict[str, models.ModelStubType]:
+    return {
+        obj.name: obj
+        for _, obj in inspect.getmembers(models)
+        if (
+            inspect.isclass(obj)
+            and issubclass(obj, models._ModelStub)
+            and obj != models._ModelStub
+        )
+    }
+
+
+def list_models() -> List[str]:
     """List available models for training."""
+    return [name for name in _list_models()]
+
+
+def list_model_options() -> Dict[str, List[Dict[str, Any]]]:
+    """List available options per model."""
+    return {
+        name: [
+            {
+                'name': parameter.name,
+                'type': parameter.annotation,
+                'required': parameter.default == parameter.empty,
+                'default': (
+                    parameter.default if parameter.default != parameter.empty else None
+                ),
+            }
+            for parameter in inspect.signature(
+                _model_class.__init__
+            ).parameters.values()
+            if parameter.name != 'self'
+        ]
+        for name, _model_class in _list_models().items()
+    }
+
+
+def describe_models() -> None:
+    """Describe available models in a table."""
     table = Table(title='Finetuner backbones')
+    header = models._ModelStub.header()
+    _no_wraps = [True] * (len(header) - 1) + [False]
```

Comment:
let's always `no_wrap=True`, i figured out it's not shwoing up correctly in jupyter notebook